payloads/extensions/linux_mount.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# LINUX_MOUNT v1 by @drapl0n
+# Auto mounts BashBunny on GNU/Linux systems.
+# NOTE: Mount path is stored in variable "lmnt".
+# Usage: LINUX_MOUNT - to automatically mount BashBunny.
+#	 LINUX_UMOUNT - to unmount mounted BashBunny.
+
+function LINUX_MOUNT() {
+	Q DELAY 1000
```

Comment:
Why have a DELAY at the beginning of the function?

Reply:
whenever Bunny is plugged in systems with desktop environment, it takes a second to process it as storage drive and prompt it in system tray.